### DISCUSSION TITLE
Gnome panel color

### DISCUSSION THREAD

**Discussion by IamRezaMousavi** [GradienceTeam/Gradience#767] [Q&A] [answered]
How can set panel color like other GTK apps??
![Screenshot from 2023-05-14 17-51-59](https://github.com/GradienceTeam/Gradience/assets/83984878/2ef1e765-f229-44bd-a246-e8e83c9e72cd)
![Screenshot from 2023-05-14 17-52-09](https://github.com/GradienceTeam/Gradience/assets/83984878/cd6c64e5-866d-4227-9cd3-5abe5f5079d6)

**Comment by daudix** [comment, marked as answer]
This feature is implemented and will be available in next release, but you can use it already by installing [nightly build](https://nightly.link/GradienceTeam/Gradience/workflows/build/main/gradience-devel-x86_64.zip)

**Reply by IamRezaMousavi** [reply]
I used `yay -S gradience-git`

**Reply by daudix** [reply]
Hm, then it should be already available, check out "Theming" tab and apply shell theme

_of course you need to update it to latest version_

**Reply by IamRezaMousavi** [reply]
Yes It's work.
Thank you :)
